tests/dynamic_honey_badger.rs

```diff
@@ -75,8 +77,8 @@ where
             .iter()
             .filter(|(_, node)| {
                 node_busy(*node)
-                    && !node.instance().has_input()
-                    && node.instance().netinfo().is_validator()
+                    && !node.instance().algo().has_input()
+                    && node.instance().algo().netinfo().is_validator()
```

Comment:
Maybe we should add these as convenience methods to `SenderQueue` (`has_input` only for `SenderQueue<DHB>` and `SenderQueue<HB>`, of course).